tests/test_logic.py

```diff
@@ -1483,3 +1484,19 @@ def test_rename_tab_avoid_duplicating_other_tab_name():
                                               'A file of that name is already '
                                               'open in Mu.')
     assert mock_tab.path == 'old.py'
+
+def test_logic_independent_import_logic():
```

Comment:
Looks like pystyle is complaining about not having 2 blank lines between functions: `E302 expected 2 blank lines, found 1`.
Same for the one below.

Reply:
(Sigh; once again a reply hasn't made it through to GH)

Thanks; good catch. In short: I didn't run make check. I'll go back and rejig.

BTW I've never encountered a codebase which actually puts two blank lines between functions (except possibly to mollify nagging linters...)